sites/page-details/index.json-ld

```diff
@@ -110,12 +110,12 @@
 					"fr": { "href": "page-details-pft-fr.html", "text": "Détails de la page avec Outil de rétroaction sur la page" }
 				},
 				{
-					"en": { "href": "demoted/page-details-v3-en.html", "text": "Demoted - Page details version 3.0" },
-					"fr": { "href": "demoted/page-details-v3-fr.html", "text": "Rétrogradé - Détails de la page version 3.0" }
+					"en": { "href": "deprecated/page-details-v3-en.html", "text": "Deprecated - Page details version 3.0" },
+					"fr": { "href": "deprecated/page-details-v3-fr.html", "text": "Obsolète - Détails de la page version 3.0" }
```

Comment:
Let's add the GCWeb version of when we deprecated that pattern. It's to ease future clean up 

```suggestion
					"en": { "href": "deprecated/page-details-v3-en.html", "text": "Deprecated as GCWeb v16.0.0 - Page details version 3.0" },
					"fr": { "href": "deprecated/page-details-v3-fr.html", "text": "Obsolète à partir de GCWeb v16.0.0 - Détails de la page version 3.0" }
```